packages/bierzo-wallet/src/logic/account.unit.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { ChainId } from '@iov/bcp';
+
+import { withChainsDescribe } from '../utils/test/testExecutor';
+import { lookupRecipientAddressByName } from './account';
+import { disconnect } from './connection';
+
+withChainsDescribe('Logic :: account', () => {
+  afterAll(async () => {
+    await disconnect();
```

Comment:
This `disconnect()` was changed to synchronous. So no `async` and `await` needed